packages/nextjs/hooks/scaffold-eth/useAccountBalance.ts

```diff
@@ -19,7 +19,9 @@ export function useAccountBalance(address?: string) {
   });
 
   const onToggleBalance = useCallback(() => {
-    setIsEthBalance(!isEthBalance);
+    if (price > 0) {
+      setIsEthBalance(!isEthBalance);
+    }
   }, [isEthBalance]);
```

Comment:
I think we should add a `price` dependency here because even eslint sets the warning and makes sense to add it as a dependency

Reply:
Yup we should add it.

BTW just curious how did you get the eslint warning as in on running `yarn next:lint` it returned `No ESLint warnings or errors` for me?

Reply:
My IDE was complaining :   
![Screenshot 2023-04-28 at 12 51 32 PM](https://user-images.githubusercontent.com/80153681/235081614-3377dee0-ba07-4fa6-82ca-84ba23d61584.jpg)

Reply:
Ohhh its the eslint extension on VSC, I did have it installed thanks